dts/bindings/adc/raspberrypi,pico-adc.yaml

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2022 TOKTIA Hrioshi <tokita.hiroshi@fujitsu.com>
+# SPDX-License-Identifier: Apache-2.0
+
+description: RaspberryPi Pico ADC
+
+compatible: "raspberrypi,pico-adc"
+
+include: [adc-controller.yaml, pinctrl-device.yaml, reset-device.yaml]
+
+properties:
+    interrupts:
+      required: false
```

Comment:
Why is this here? It looks unnecessary to me.

Reply:
It's my careless. Removing it.
I implemented without interrupts firstly, and I forgot to remove it.

Reply:
Thanks!